lib/ansible/module_utils/vmware.py

```diff
@@ -1176,44 +1176,6 @@ def get_all_port_groups_by_host(self, host_system):
             pgs_list.append(pg)
         return pgs_list
 
-    def find_network_by_name(self, network_name=None):
```

Comment:
I think we should debug this method rather than removing it altogether. And going forward add and use methods from `Pyvmomi` class.

Reply:
Hi @Tomorrow9, could you take a look at the issue? I don't mind closing this PR, but we need to land a fix for this issue.

Reply:
@Akasurde has found the issue, my last iteration of this PR address it.